qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -91,38 +88,10 @@ def _choose_layout_condition(property_set):
             coupling_map, max_iterations=1, seed=seed_transpiler, swap_trials=5
         )
 
-    toqm_pass = False
     # Choose routing pass
-    # TODO: Remove when qiskit-toqm has it's own plugin and we can rely on just the plugin interface
-    if routing_method == "toqm" and "toqm" not in list_stage_plugins("routing"):
-        HAS_TOQM.require_now("TOQM-based routing")
-        from qiskit_toqm import ToqmSwap, ToqmStrategyO0, latencies_from_target
-
-        if initial_layout:
-            raise TranspilerError("Initial layouts are not supported with TOQM-based routing.")
-
-        toqm_pass = True
-        # Note: BarrierBeforeFinalMeasurements is skipped intentionally since ToqmSwap
-        #       does not yet support barriers.
```

Comment:
Is this handling disappearing going to cause any issues?

Reply:
This is handled within TOQM now when generating the routing pass manager, e.g. `common.generate_routing_passmanager(..., use_barrier_before_measurement=False)`.

(I think that option was probably added because of TOQM 😄)

Reply:
Ok, sounds good to me in that case!